### PR TITLE
Update license.txt

### PR DESCRIPTION
**Pull request by joeriddles**
Fill out the required fields in the license.txt file.

### CODE REVIEW

**Review by GrantErickson — COMMENTED** [review]
Seems ok. Wish this was something we could update automatically, but I don't know of a way.

**Review by GrantErickson — APPROVED** [review]

**Comment by adamskt** [comment]
You can automate this with a GitHub action: https://github.com/marketplace/actions/update-license-copyright-year-s

File: license.txt [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2021 IntelliTect
```
Comment (adamskt) [inline license.txt:189]: I don't know how much it may matter legally, but the company's name (at least according to my W-2) is still "IntelliTechture"

**Comment by joeriddles** [comment]
It's not that the year was out of date, the year and copyright owner info were never filled out to begin with. That GitHub action looks cool.

File: license.txt (hunk 1 above)
Comment (adamskt) [reply license.txt:189]: image

**Comment by ascott18** [comment]
@adamskt @GrantErickson updating the year on copyright notices is generally unnecessary and in some cases can actually be more legally disadvantageous than just leaving it alone. 

See https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

**Review by ascott18 — APPROVED** [review]

File: license.txt (hunk 1 above)
Comment (ascott18) [reply license.txt:189]: I'd be very surprised if we didn't have a DBA for the IntelliTect name, which I'd think would be legally sufficient in this case.

**Pull request merged**